specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2017-07-01/bms.json

```diff
@@ -1745,6 +1745,10 @@
           "items": {
             "$ref": "#/definitions/SubProtectionPolicy"
           }
+        },
```

Comment:
adding new writable properties requires a new api-version.

Reply:
Hi. Can you please elaborate on why it needs new api-version? This doesn't break existing clients because this is an optional parameters and null is the default value.

Reply:
https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#123-definition-of-a-breaking-change
paragraph 2 sentence 2. We are just trying to enforce what the architects have laid out as definition of a breaking change for Azure.

Reply:
This change is removed in this PR and will be added with new api version in later PR.